packages/types/index.d.ts

```diff
@@ -131,6 +131,11 @@ export interface IStep {
 export interface IFlowConfig {
   maxQps?: number
   rejectIfOverMaxQps?: boolean
+  errorConfig?: IFlowErrorConfig
+}
+
+export interface IFlowErrorConfig {
+  notificationFrequency?: 'once' | 'always'
```

Comment:
Can this be required? Not seeing a use case for this to be optional.

```suggestion
  notificationFrequency: 'once' | 'always'
```

Reply:
The main reasoning being that `errorConfig` must be optional (since existing pipes will not have it), but if anyone sets `errorConfig` then they must also set `notificationFrequency`

Reply:
anw i used `once_per_day` instead cus graphql enum don't accept hypens hmm